Source/DafnyCore/Verifier/Translator.cs

```diff
@@ -5845,7 +5853,7 @@ void CheckCasePatternShape<VT>(CasePattern<VT> pat, Bpl.Expr rhs, IToken rhsTok,
         }
 
         {
-          // Requires(Ty.., F#Handle( Ty1, ..., TyN, Layer, self), Heap, arg1, ..., argN)
+          // Requires(Ty.., F#Handle( Ty1, ..., TyN, Layer, reveal, self), Heap, arg1, ..., argN)
```

Comment:
Based on my initial experiments, this should make it easier to allow all functions to have a reveal parameter. The fact that `requires` didn't have this parameter was an awkward asymmetry.